src/Python/qsharp-core/qsharp/clients/iqsharp.py

```diff
@@ -242,9 +245,11 @@ def _execute(self, input, return_full_result=False, raise_on_stderr : bool = Fal
 
         def log_error(msg):
             errors.append(msg)
-
+        
         handlers = {
-            'execute_result': (lambda msg: results.append(msg))
+            'execute_result': (lambda msg: results.append(msg)),
+            'render_execution_path':  (lambda msg: results.append(msg)),
+            'display_data': lambda msg: ...
```

Comment:
Is the `...` here intended?

Reply:
Yes, it's the equivalent of a no-op. If I don't include it the default handler of display_data is to print a message on the screen.